USER_GUIDE.md

```diff
@@ -283,3 +286,91 @@ func main() {
 }
 
 ```
+
+#### AWS SDK V2 with OpenSearch Serverless
```

Comment:
We should merge the managed and serverless portions, these are the same now no?

Reply:
Combined them.